### PR TITLE
add information about the daemon

### PR DESCRIPTION
**Pull request by guzzisti**
In #7326 we found that the docs are a bit outdated regarding the frontend worker. I removed the mention from the install docs.

### CODE REVIEW

File: doc/Install.md [hunk 1]
```diff
@@ -249,10 +250,20 @@ You can generally find the location of PHP by executing "which php".
...
-If it is not possible to set up a cron job then please activate the "frontend worker" in the administration interface.
+#### worker alternative: daemon
+Otherwise, you’ll need to use the command line on your remote server and start the Friendica daemon (background task) using the following command:
+
+    cd /path/to/friendica; php bin/daemon.php start
+Once started, you can check the daemon status using the following command:
+
+    cd /path/to/friendica; php bin/daemon.php status
+
+After a server restart or any other failure, the daemon needs to be restarted. This could be achieved by a cronjob.
```
Comment (MrPetovan) [inline doc/Install.md]: Actually, please add a line feed before each new sentence (after the period). This is an internal convention to make diffs nicer when editing Markdown files.

File: doc/de/Install.md [hunk 2]
```diff
@@ -219,7 +219,18 @@ Du kannst den PHP-Pfad finden, indem du den Befehl „which php“ ausführst.
...
+#### worker alternative: daemon
+Wenn Du keine Möglichkeit hast, Cronjobs anzulegen, kannst Du den Friendica Daemon als Hintergrund Prozess über die Kommandozeile deines Servers starten:
+
+    cd /path/to/friendica; php bin/daemon.php start
+
+Sobald er gestartet wurde, solltest Du überprüfen ob der Daemon wirklich läuft:
+
+    cd /path/to/friendica; php bin/daemon.php status
+
+Du solltest sicherstellen, dass der Daemon bei z.B. einem Serverneustart automatisch neu gestartet wird oder im Fehlerfall. Dies kann über z.B. einen Cronjob geschehen.
```
Comment (MrPetovan) [inline doc/de/Install.md]: Same here about one sentence per line.

**Comment by guzzisti** [comment]
Sorry for the commit mess...i mixed up branches... :/

**Comment by MrPetovan** [comment]
It's ok, I'm still waiting for a German-reading reviewer.

**Comment by guzzisti** [comment]
Is it intentional that there are two different install.mds in root and doc directory with differing content and formatting?
if you are interested i'll come up with a another PR later on to merge the content of those files.

**Comment by MrPetovan** [comment]
Yes, one is displayed nicely in GitHub (root install.md), while the other is displayed in Friendica (/help/install on any node). If there are any obsolete information in any of these files, they need to be updated, but both are meant to stay.

**Comment by guzzisti** [comment]
ok, got it...in my opinion keeping duplicate information is unfortunate and prone to diverging content as it is now (see troubleshooting section for example).
As both files are MarkDown i guess it should be possible to have a single file serve both purposes. I can't find big differences in displaying this files on github.

**Comment by MrPetovan** [comment]
I mean it is an expected file in the root directory on GitHub.

**Comment by guzzisti** [comment]
Is it? I'm not a hardcore github user...i only know about the `readme.md` which is expected and displayed on the project page.
The Readme should point the user to additional docs if necessary.

**Comment by MrPetovan** [comment]
If you know what you're doing, please make it so.

**Comment by annando** [comment]
Technically the frontend worker still is available, though it is really not recommend and will most likely cause problems with updates.

**Comment by MrPetovan** [comment]
Then we should stop talking about it.

**Comment by guzzisti** [comment]
@annando The frontend worker wasn't working atleast for @jranma and me.
If it's not recommended anymore, i think it's better to remove it from the docs and afterwards remove the implementation as well.

@MrPetovan I'll have a look at the docs and will propose some changes in a separate PR which can be discussed there.

**Comment by annando** [comment]
I would like to have an additional opinion to this. @tobiasd what do you think?

**Comment by guzzisti** [comment]
This PR is outdated, feel free to comment on #7366

**Comment by tobiasd** [comment]
I think it is a really sad day that Friendica drops the poorman cron/frontend worker. It was never recommended but it always worked. So it is really sad that you yanked it out of usability.

**Comment by guzzisti** [comment]
I recommend to continue the discussion if it's worth to fix the frontend worker in #7326

**Comment by tobiasd** [comment]
Possible true, but the github guided me to this thread here as annando mentioned me here ;-)

**Comment by annando** [comment]
Can we discuss this possibly in the developer forum? I really would like to show why the frontend worker (and poormancron) never really worked.

**Comment by tobiasd** [comment]
As I used it myself, I know that it worked well enough to keep a node running ;-) But I agree, the issue tracker is no place for discussions.

**Comment by annando** [comment]
I wrote some lines about this: https://forum.friendi.ca/display/ec054ce7-145d-2d72-34b3-499176629383

**Pull request closed without merging**